### DISCUSSION TITLE
Mph vs Kmh

### DISCUSSION THREAD

**Discussion by michaelsidenius** [alandtse/tesla#457] [Q&A] [answered]
I get Mph in charging speed even my settings are Metric in HA. How can I set to Kmh (metric)?

**Comment by carleeno** [comment, marked as answer]
I'm not sure why it didn't default to your preference upon first install, but you can change the unit of measurement of any sensor at any time by clicking on that sensor, clicking settings, and changing the unit of measurement.

**Reply by michaelsidenius** [reply]
Thanks! That did the trick!
